plugins/scaffolder-backend/src/scaffolder/actions/builtin/publish/gitlabMergeRequest.ts

```diff
@@ -170,15 +165,15 @@ export const createPublishGitlabMergeRequestAction = (options: {
 
       try {
         const mergeRequestUrl = await api.MergeRequests.create(
-          ctx.input.projectid,
+          projectPath,
           destinationBranch,
           String(defaultBranch),
           ctx.input.title,
           { description: ctx.input.description },
         ).then((mergeRequest: { web_url: string }) => {
           return mergeRequest.web_url;
         });
-        ctx.output('projectid', ctx.input.projectid);
+        ctx.output('project path: ', projectPath);
```

Comment:
this should be

```
ctx.output('projectid', projectPath);
ctx.output('projectPath', projectPath);
```

Reply:
Let's keep the existing output but deprecate it, and then we can remove it when we make some other breaking changes.